jobs/regular/discourse_post_event/bump_topic.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DiscoursePostEventBumpTopic < ::Jobs::Base
+    sidekiq_options retry: false
+
+    def execute(args)
+      raise Discourse::InvalidParameters.new(:topic_id) if args[:topic_id].blank?
+      raise Discourse::InvalidParameters.new(:date) if args[:date].blank?
+
+      puts "run job"
+      topic = Topic.find_by(id: args[:topic_id].to_i)
+      event_user = User.find_by(id: topic.user_id)
```

Comment:
Could `topic.user_id` or the `#find_by` return nil here?

Reply:
I think so, but that case is already handled inside `set_or_create_timer`